src/tools/nbench/nbench.cpp

```diff
@@ -85,7 +85,7 @@ multimap<size_t, string> aggregate_timing_details(const vector<PerfShape>& perf_
     }
 
     multimap<size_t, string> rc;
-    for (const pair<string, size_t>& t : timing)
+    for (auto& t : timing)
```

Comment:
Can this be `const auto&`? I much prefer to keep the const

Reply:
`auto` can't make `const`ness go away.